### PR TITLE
[ECO-819] Enums

### PR DESCRIPTION
**Pull request by hernantain**
**What:** Extracting strings into enums for reusability along handlers and models.

**Why:** Same strings are repeated along all over the code and is error prone.

**How:** Creating enums for most 'entities' fields.

https://trustar.atlassian.net/browse/ECO-819

### CODE REVIEW

File: trustar2/handlers/indicators.py [hunk 1]
```diff
@@ -88,7 +97,7 @@ def search(self):
         return (
             self.create_query(Methods.POST)
             .set_params(self.payload_params)
-            .set_query_string({"pageSize": self.query_params.get("pageSize", 25)})
+            .set_query_string({PAGE_SIZE: self.query_params.get(PAGE_SIZE, 25)})
```
Comment (marcos-vivar) [inline trustar2/handlers/indicators.py:100]: @hernantain Do you feel like we can parameterize the "25" to a DEFAULT_PAGE_SIZE value or something like that?

**Review by manuelramos — APPROVED** [review]
Looks good! :100: :1st_place_medal:

**Review by marcos-vivar — APPROVED** [review]
@hernantain impressive refactor and parameterization! 🏆 

I left one suggestion that's not blocking this great effort! :shipit:

File: trustar2/handlers/indicators.py [hunk 2]
```diff
@@ -47,35 +56,35 @@ def _get_entity(entity, entity_type):
     def set_priority_scores(self, scores):
         if not isinstance(scores, list) or any([s for s in scores if s > 3 or s < -1]):
             raise AttributeError("scores should be a list of integers between -1 and 3")
-        self.set_payload_param("priorityScores", scores)
+        self.set_payload_param(IndicatorEnum.PRIORITY_SCORES.value, scores)
```
Comment (imonaco) [inline trustar2/handlers/indicators.py:59]: We are using constants declared in the top of the file, and we are also using the .value directly , like here. Is there any reason of using both behaviors? If not, may be it be better to just stick to  one way ?

File: trustar2/handlers/indicators.py (hunk 1 above)
Comment (imonaco) [reply trustar2/handlers/indicators.py:100]: Good suggestion @marcos-vivar  !

**Review by imonaco — APPROVED** [review]
Just some minor comments, but great job @hernantain  !! 💯

**Pull request merged**